jovian/callbacks/fastai_callback.py

```diff
@@ -0,0 +1,54 @@
+from fastai.basic_train import Learner
```

Comment:
you can add `__init__.py` in callbacks folder and import these callbacks there. so we can do something like:
`import JovianFastaiCallback from jovian.callbacks`

Reply:
i think a blank `__init__.py` will satisfy that. `jovian.utils' has a blank init too.
Doing that imports will throw if one of the packages is not installed `keras` or `fastai` ?

Reply:
@sidujjain if a user has installed only keras or fastai, the other dependencies throws error of not being found.